lib/utils/moment/src/index.js

```diff
@@ -24,4 +24,25 @@ moment.toYYYYMM = (period) => {
   return time.getUTCFullYear() * 100 + time.getUTCMonth() + 1;
 };
 
+//
+// Returns a timestamp (in milliseconds since epoch), at the specified
+// year, month, day, hour, minute, second, and milliseconds.
+//
+moment.utcTimestamp = (year, month, day, hour, minute, second, millis) => {
+  return moment.utc([year, month, day, hour, minute, second, millis]).valueOf();
+};
+
+moment.Jan = 0;
```

Comment:
This can be transformed to `monthNumber = (monthName) => moment().month(monthName).format("M")` ?

Reply:
We prefer to use constants.